scripts/shared/bump-version.ts

```diff
@@ -10,6 +12,8 @@ export function bumpVersion(version: string, mode: TuiReleaseMode): string {
             return `${major}.${++minor}.0`;
         case 'patch':
             return `${major}.${minor}.${++patch}`;
+        case 'prerelease':
+            return `${alpha === -1 ? ++major : major}.0.0-${++alpha}`;
```

Comment:
Should we add word `alpha` in the version ?

For example, `3.0.0-alpha.3` or `3.0.0-beta.1` ?
Or `3.0.0-3` is enough  for our needs ?

Reply:
as you wish, so be it